modules/ROOT/pages/organization-overview.adoc

```diff
@@ -74,7 +74,7 @@ See xref:custom-user-information-in-bonita-bpm-studio.adoc[custom user informati
 
 === In Bonita Portal:
 
-This custom information can only be seen and xref:custom-user-information-in-bonita-bpm-portal.adoc[edited] by users who have the Administrator profile.
+This custom information can only be seen and xref:custom-user-information-in-bonita-applications.adoc[edited] by users who have the Administrator profile.
```

Comment:
custom-user-information-in-bonita-bpm-applications.adoc instead of custom-user-information-in-bonita-applications.adoc as we are replacing custom-user-information-in-bonita-bpm-portal.adoc

Reply:
Done